src/test/java/org/heigit/ohsome/ohsomeapi/controller/PostControllerTest.java

```diff
@@ -112,9 +112,20 @@ public void nonNodedLinestringsIntersectionTest() {
   }
 
   /*
-   * false parameter tests
+   * false parameter and no parameters tests
    */
 
+  @Test
+  public void queryWithoutParametersTest() {
+    TestRestTemplate restTemplate = new TestRestTemplate();
+    ResponseEntity<JsonNode> response =
+            restTemplate.postForEntity(server + port + "/elements/perimeter", null,
```

Comment:
the indentation here is different from e.g. [line 135 below](https://github.com/GIScience/ohsome-api/pull/233/files#diff-5738da37ce58a90b3ab106a9ff11247cf4cad7fa136f4fc45233a94d5dedbc08R135). Maybe the rest of the statement (`JsonNode.class);`) would otherwise fit into a single line?